Sources/PopupBridge/POPPopupBridge.swift

```diff
@@ -1,63 +1,72 @@
 import Foundation
-import SafariServices
 import WebKit
+import AuthenticationServices
 
 @objcMembers
-public class POPPopupBridge: NSObject, WKScriptMessageHandler, SFSafariViewControllerDelegate {
+public class POPPopupBridge: NSObject, WKScriptMessageHandler {
+
+    /// Exposed for testing
+    var returnedWithURL: Bool = false
     
     // MARK: - Private Properties
     
     private let messageHandlerName = "POPPopupBridge"
-    private let hostName = "popupbridgev1"
-    
+    private let hostName = "popupbridgev1"    
     private let webView: WKWebView
-    private let urlScheme: String
     private let delegate: POPPopupBridgeDelegate
-    private var safariViewController: SFSafariViewController?
+    private var webAuthenticationSession: WebAuthenticationSession
     
-    private static var returnBlock: ((URL) -> Bool)?
+    private var returnBlock: ((URL) -> Void)? = nil
```

Comment:
It doesn't seem like there is a specific reason we need to return a bool but I may have missed a use case for this. Everything is working as expected in the demo app. The only exception is the Ideal flow. Once this is merged we should update the [example here to pass in the updated url scheme](https://github.com/braintree/popup-bridge-example/blob/master/local-payment-methods.html#L78).

Reply:
> Once this is merged we should update the [example here to pass in the updated url scheme](https://github.com/braintree/popup-bridge-example/blob/master/local-payment-methods.html#L78).

The JS integration shouldn't care about a URL scheme anymore, though for the new ASWeb version of PopupBridge?

Reply:
> It doesn't seem like there is a specific reason we need to return a bool but I may have missed a use case for this.

The bool is returned to the merchant after they call[ `open(url:)` ](https://github.com/braintree/popup-bridge-ios/blob/844157c1de77834987c9aca32c4d3c79fc821cb9/Sources/PopupBridge/POPPopupBridge.swift#L56-L62), which looks to be removed in this PR. We should call out this method was removed in the CHANGELOG.

Reply:
> The JS integration shouldn't care about a URL scheme anymore, though for the new ASWeb version of PopupBridge?

Yep, but since we have a fallback URL hardcoded it's always attempting to fall back to that URL scheme, which doesn't exist. We _should_ be able to remove it completely but will also need to make sure Android popup bridge doesn't rely on that being hardcoded.

Reply:
> The bool is returned to the merchant after they call[ open(url:) ](https://github.com/braintree/popup-bridge-ios/blob/844157c1de77834987c9aca32c4d3c79fc821cb9/Sources/PopupBridge/POPPopupBridge.swift#L56-L62), which looks to be removed in this PR. We should call out this method was removed in the CHANGELOG.

Ah, do we need an additional line? I had assumed this would be covered by the [following line already in the CHANGELOG](https://github.com/braintree/popup-bridge-ios/blob/master/CHANGELOG.md#unreleased): `Remove deprecated POPPopupBridge.open(url:sourceApplication:) & POPPopupBridge.open(url:options:) methods`. It didn't look like we had called out the rename to `open(url:)` anywhere but happy to add an additional line if it's not clear!

Reply:
Ah 🤦  - I thought it was a rename and not 3 separate methods. Updated here: https://github.com/braintree/popup-bridge-ios/pull/57/commits/38eede3be39a3e165e2e24368a545fa00a612bfa